engine/base/runtime/src/main/scala/pl/touk/nussknacker/engine/baseengine/metrics/NodeCountingListener.scala

```diff
@@ -0,0 +1,40 @@
+package pl.touk.nussknacker.engine.baseengine.metrics
+
+import cats.data.NonEmptyList
+import pl.touk.nussknacker.engine.api.exception.EspExceptionInfo
+import pl.touk.nussknacker.engine.api.runtimecontext.EngineRuntimeContext
+import pl.touk.nussknacker.engine.api.{Context, EmptyProcessListener, Lifecycle, MetaData}
+import pl.touk.nussknacker.engine.util.exception.ExceptionRateMeter
+import pl.touk.nussknacker.engine.util.metrics.{Counter, InstantRateMeterWithCount, MetricIdentifier, MetricsProviderForScenario, RateMeter}
+
+class NodeCountingListener extends EmptyProcessListener with Lifecycle with ExceptionRateMeter {
```

Comment:
Can `ExceptionRateMeter` be composed instead of inherited?

Reply:
:+1: